### PR TITLE
Removes odometer covariance from public API.

### PR DESCRIPTION
**Pull request by damonkohler**

### CODE REVIEW

File: cartographer/mapping_3d/proto/local_trajectory_builder_options.proto [hunk 1]
```diff
@@ -57,4 +57,7 @@ message LocalTrajectoryBuilderOptions {
       kalman_local_trajectory_builder_options = 10;
   optional OptimizingLocalTrajectoryBuilderOptions
       optimizing_local_trajectory_builder_options = 11;
+
+  optional double odometer_translational_variance = 13;
```
Comment (wohe) [inline cartographer/mapping_3d/proto/local_trajectory_builder_options.proto]: These are unused, right? Remove?

File: configuration_files/trajectory_builder_3d.lua [hunk 2]
```diff
@@ -83,6 +83,9 @@ TRAJECTORY_BUILDER_3D = {
       translation_delta_cost_weight = 1e-1,
       rotation_delta_cost_weight = 1e-1,
     },
+
+    odometer_translational_variance = 1e-7,
```
Comment (wohe) [inline configuration_files/trajectory_builder_3d.lua]: This should move to the kalman_local_trajectory_builder?

File: cartographer/mapping_3d/proto/local_trajectory_builder_options.proto (hunk 1 above)
Comment (damonkohler) [inline cartographer/mapping_3d/proto/local_trajectory_builder_options.proto]: done.

File: configuration_files/trajectory_builder_3d.lua (hunk 2 above)
Comment (wohe) [inline configuration_files/trajectory_builder_3d.lua]: Misread the curly braces.

**Pull request merged**